userspace/falco/falco.cpp

```diff
@@ -873,21 +877,28 @@ int falco_init(int argc, char **argv)
 		}
 		else
 		{
-			conf_stream.open(FALCO_SOURCE_CONF_FILE);
+
+#ifdef BUILD_TYPE_RELEASE
```

Comment:
From my understanding:
- If `BUILD_TYPE_RELEASE` is defined, we first try to load from `FALCO_INSTALL_CONF_FILE`, and then from `FALCO_SOURCE_CONF_FILE`
- If `BUILD_TYPE_RELEASE` is not defined, we only try to load from `FALCO_SOURCE_CONF_FILE`

Which is not the semantic we seem to describe in the `-help` section. Am I missing something?

Reply:
No worries!

Reply:
Thank you :pray: